src/ApprenticeAan/SFA.DAS.ApprenticeAan.Application/LeavingReasons/Queries/GetLeavingReasons/GetLeavingReasonsQueryHandler.cs

```diff
@@ -0,0 +1,21 @@
+﻿using MediatR;
+using SFA.DAS.ApprenticeAan.Application.Infrastructure;
+using SFA.DAS.ApprenticeAan.Application.Model;
+
+namespace SFA.DAS.ApprenticeAan.Application.LeavingReasons.Queries.GetLeavingReasons;
+
+public class GetLeavingReasonsQueryHandler : IRequestHandler<GetLeavingReasonsQuery, List<LeavingCategory>>
```

Comment:
I am discouraging use of Handlers if it is as simple as returning the results from inner api. You can just have controller using outer api client directly. Handler should have a purpose, in this case it has none.